app/Http/Controllers/FeedBackController.php

```diff
@@ -0,0 +1,334 @@
+<?php
+
+namespace Proto\Http\Controllers;
+
+use Exception;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Mail;
+use Illuminate\Support\Facades\Redirect;
+use Illuminate\Support\Facades\Session;
+use Illuminate\View\View;
+use Proto\Mail\FeedbackReplyEmail;
+use Proto\Models\Feedback;
+use Proto\Models\FeedbackCategory;
+use Proto\Models\FeedbackVote;
+use Proto\Models\User;
+
+class FeedBackController extends Controller
+{
+    /**
+     * @param string $category
+     * @return View
+     */
+    public function index(Request $request, String $category): View
+    {
+        $category = FeedbackCategory::where('url', $category)->firstOrFail();
+        $mostVoted = $this->getMostVoted($category);
+
+        $unreviewed = $this->getUnreviewed($category);
+        return view('feedbackboards.index', ['data' => $this->getFeedbackQuery($category)->paginate(20), 'mostVoted' => $mostVoted ?? null, 'category'=>$category, 'unreviewed'=>$unreviewed]);
+    }
+
+    private function getFeedbackQuery(FeedbackCategory $category): HasMany
+    {
+        $feedback = $category->feedback()->orderBy('created_at', 'desc')->with('votes');
+        if($category->review && Auth::user()->id !== $category->reviewer_id) {
+            $feedback = $feedback->where('reviewed', true);
+        }
+        return $feedback;
+    }
+
+    private function getMostVoted(FeedbackCategory $category)
+    {
+        //find the most voted idea
+        $mostVotedID = FeedbackVote::query()
+            ->whereHas('feedback', function ($query) use ($category) {
+                $query->where('feedback_category_id', $category->id);
+            })
+            ->selectRaw('feedback_id, sum(vote) as votes')
+            ->groupBy('feedback_id')
+            ->having('votes', '>=', 0)
+            ->orderBy('votes')
+            ->first();
+        $mostVoted = Feedback::where('id',$mostVotedID ? $mostVotedID->feedback_id : null)->first();
```

Comment:
Nope, because I need the feedbackID, I guess in the new laravel version we can do it with  $mostVotedID?->feedback_id but this was written before that :(